runtime/worker.rs

```diff
@@ -293,6 +293,15 @@ impl MainWorker {
     Ok(())
   }
 
+  /// See [JsRuntime::execute_script](deno_core::JsRuntime::execute_script)
+  pub fn execute_script_with_return(
```

Comment:
I suggest you change `execute_script` function to return a value instead of adding a separate API for this